settings.sample.ini

```diff
@@ -0,0 +1,49 @@
+; This is a sample settings file. Copy it to "settings.ini" and configure it
+; before running make.
+
+; To load variables in bash use:
+; > source $current_dir/common.sh
+; > declare -A LIST_OF_APPS
+; > get_settings_section settings.ini fdroid_apps LIST_OF_APPS
+
+
+; General settings.
+[general]
+system_app_destination=/system/priv-app ; no trailing slash.
+user_app_destination=/data/app ; no trailing slash.
+;password_file=/path/to/secret/file
+
+; Preferred snapshot version
+snapshot_version=M12
+
+
+; Installed before those downloaded from the f-droid.org repository.
+[apps]
+FDroid=https://f-droid.org/FDroid.apk
+OrfoxFennec-debug=https://guardianproject.info/builds/OrfoxFennec/latest/OrfoxFennec-debug.apk
+
+
+; Contains a list of applications to be installed.
+; If you feel adventurous use "latest" to retrieve the latest released version.
+; NOTE: Versions are locked because we want to be sure that the apps are known
+;       to be secure.
+[fdroid_apps]
+com.artifex.mupdfdemo=55
+com.csipsimple=2459
+com.fsck.k9=22002
+com.morlunk.mumbleclient=72
+com.projectsexception.myapplist.open=16
+com.xabber.androiddev=123
+de.schildbach.wallet=209
+fr.kwiatkowskiTrack=1
+indrora.atomic=19
+net.osmand.plus=197
+org.coolreader=875
+org.ethack.orwall=35
+org.mariotaku.twidere=98
+org.mozilla.fennec_fdroid=360010
```

Comment:
Ah. I guess i lucked out before when grabbing the top download url via xpath, but seems some apps have different builds for `x86` vs `armeabi-v7a`. So I guess we need to start not just the device codename in the ini file, but also the arch. And this means templates are an even better idea, as each arch will need its own lockfile if we want this to be deterministic for more than one device :)

I'm not sure storing the specific versions in the ini is the right approach. Thinking just the app name, and then create a separate lockfile that's not really meant for people, but reads from during build. Maybe a lack of lockfile could mean you want the latest. So if someone wants to build with the latest, then just `rm whatever.lock` and build. I like lockfiles for managing the assembling of packages/APKs -- They're a pretty standard pattern.